lisa/sut_orchestrator/azure/platform_.py

```diff
@@ -936,10 +966,18 @@ def _initialize_credential(self) -> None:
                 ] = azure_runbook.service_principal_client_id
             if azure_runbook.service_principal_key:
                 os.environ["AZURE_CLIENT_SECRET"] = azure_runbook.service_principal_key
+            if azure_runbook.access_token:
+                os.environ["AZURE_ACCESS_TOKEN"] = azure_runbook.access_token
 
-            credential = DefaultAzureCredential(
-                authority=self.cloud.endpoints.active_directory,
-            )
+            if "AZURE_ACCESS_TOKEN" in os.environ:
+                credential = StaticAccessTokenCredential(
+                    os.environ["AZURE_ACCESS_TOKEN"],
+                    int(time.time()) + 3600 * azure_runbook.access_token_expiry_hours,
```

Comment:
Maybe try to parse the expiration time from the token itself. So, the parameter is not needed.